tests/smoke-tests/--typescript.test.ts

```diff
@@ -86,6 +86,18 @@ for (let packageManager of SUPPORTED_PACKAGE_MANAGERS) {
         'template-registry.js.map',
       ]);
 
+      expect(distContents.hashed.length).toBe(4);
+      expect(
+        distContents.hashed
+          .filter(file => file.includes('_rollup'))
+          .map(file => file.split('.js')[1])
+      ).to.deep.equal(["", '.map'], 'the rollup helpers are emitted with a source map');
+      expect(
+        distContents.hashed
+          .filter(file => file.includes('template-only'))
+          .map(file => file.split('.js')[1])
+      ).to.deep.equal(["", '.map'], 'the template-only component is emitted with a source map');
```

Comment:
Are these additional assertions related to gts support? Or why are they in this PR?

Reply:
For the template-only file, it was previously un-asserted.

Since the import path changed (by adding an extension), I thought it might be good to assure the template-only component is in the dist output.

:shrug: